meson.build

```diff
@@ -272,8 +272,14 @@ int main() {
  return 0;
 }
 '''
+if meson.is_cross_build()
+  ieee754_float_div = meson.get_external_property('ieee754_float_div', cc.get_id() in ['gcc', 'clang'])
+  warning('Cross-building, assuming IEEE 754 division:', ieee754_float_div)
```

Comment:
I don't think we want to warn, here. If you're building with Meson's warnings turned to errors—e.g. in CI—you're going to break perfectly valid builds.

This should be a message, like:

```suggestion
  message('IEEE 754 floating-point division (cross build):', ieee754_float_div)
```
```

Reply:
Or even summary().